browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -3858,43 +3798,36 @@ TEST_F(KeyringServiceUnitTest, SolanaKeyring) {
 }
 
 TEST_F(KeyringServiceUnitTest, SignMessage) {
-  base::test::ScopedFeatureList feature_list;
-  base::FieldTrialParams parameters;
-  parameters[features::kCreateDefaultSolanaAccount.name] = "false";
-
-  std::vector<base::test::FeatureRefAndParams> enabled_features;
-  enabled_features.emplace_back(features::kBraveWalletSolanaFeature,
-                                parameters);
-
-  feature_list.InitWithFeaturesAndParameters(enabled_features, {});
-
   KeyringService service(json_rpc_service(), GetPrefs(), GetLocalState());
   ASSERT_TRUE(RestoreWallet(&service, kMnemonic1, "brave", false));
   base::RunLoop().RunUntilIdle();
 
-  const std::vector<uint8_t> message = {0xde, 0xad, 0xbe, 0xef};
-
-  // solana keyring doesn't exist yet
-  EXPECT_TRUE(service
-                  .SignMessageBySolanaKeyring(
-                      "BrG44HdsEhzapvs8bEqzvkq4egwevS3fRE6ze2ENo6S8", message)
-                  .empty());
```

Comment:
Dropped this as we should test default behavior
Actually should remove all these enabled features and params